src/app/home/home.page.ts

```diff
@@ -1,12 +1,42 @@
 import { Component } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
 
 @Component({
   selector: 'app-home',
   templateUrl: 'home.page.html',
   styleUrls: ['home.page.scss'],
 })
 export class HomePage {
+  profileForm: FormGroup;
+  nombre: string;
+  apellido: string;
+  email: string;
+  password: string;
+  
+  constructor() {
+    this.profileForm = new FormGroup({
+      nombreFormControl: new FormControl('', [
+        Validators.required,
+        Validators.pattern('[a-zA-Z ]+')
+      ]),
+      apellidoFormControl: new FormControl('', [
+        Validators.required,
+        Validators.pattern('[a-zA-Z ]+')
+      ]),
+      emailFormControl: new FormControl('', [
+        Validators.required,
+        Validators.email
+      ]),
+      passwordFormControl: new FormControl('', [
+        Validators.required,
+        Validators.minLength(8),
+        Validators.maxLength(12),
```

Comment:
El maximo es 32

Reply:
https://www.concretepage.com/angular-2/angular-2-4-minlength-and-maxlength-validation-example#Template-driven
checa este link